fix(multipe): don't use div containers inside labels

mhaertwig opened the pull request:
According to the specification, div containers are not allowed inside labels.

Closes #1619

kyubisation reviewed (approved):
LGTM 👍 
Should we maybe add a note to the accessibility considerations of the panels to avoid block elements?

mhaertwig commented:
> LGTM 👍 Should we maybe add a note to the accessibility considerations of the panels to avoid block elements?

Good idea, yes. I've updated the docs and added some tests to ensure `div`'s don't come back.

jeripeierSBB reviewed (approved):
LGTM
